include/TradingEngine/order_status.h

```diff
@@ -0,0 +1,22 @@
+/*
+Contains declaration for OrderStatus enum and Order status map. 
+OrderStatus enum is a property of the Order class and Order
+Status map is used for string representation of Order info.
+*/
+
+#ifndef TRADING_ENGINE_INCLUDE_ORDER_STATUS_H_
```

Comment:
btw, how do you come up with these names and why do they have _ at the end?

Reply:
I beleive I got this from the google C++ style guide, under the link called "The #define Guard'